<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMain">
     <attribute name="title">
      <string>主页面</string>
     </attribute>
     <widget class="QPushButton" name="btnListen">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>监听</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPort">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2345</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboxIP">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabP500">
     <attribute name="title">
      <string>恒流泵</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>681</width>
        <height>341</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>设备状态</string>
      </property>
      <widget class="QPushButton" name="btnSearch">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>270</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>运行状态：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最大压力：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最小压力：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备流量：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>实时压力：</string>
       </property>
      </widget>
      <widget class="QLabel" name="labStatus">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labFlowrate">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labPress">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>160</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labMaxpress">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labMinpress">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>280</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>681</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>控制模块</string>
      </property>
      <widget class="QPushButton" name="btnRun">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>180</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>流量：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtMinPress">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>180</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtFlowRate">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最大压力：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ml/min</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MPa</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>180</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MPa</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最小压力：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtMaxPress">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
